File: haskell/haskellFB_win/fizz_buzz.hs
--runs through a set of numbers
--and prints Fizz if divisible by 3
--Buzz if divible by 5 and FizzBuzz when divisble by both
--run in command line

fbAppendValue :: Int -> String
fbAppendValue x
    | (x `mod` 3 == 0) && (x `mod` 5 == 0) = "FizzBuzz"
    | x `mod` 3 == 0 = "Fizz"
    | x `mod` 5 == 0 =  "Buzz"
    |otherwise = show x

fizzBuzz :: String -> [String]
fizzBuzz x = map(fbAppendValue) inputList
    where inputList = [1..(read(x) :: Int)]

main :: IO ()
main = do
    putStrLn "Hello. Please input the max number for your fizzbuzz"
    numberToFB <- getLine
    mapM_ print (fizzBuzz numberToFB)

File: python/fizz_buzz.py
#for input n, print a list of
#fizz buzz that is n long.
def fizz_buzz (n):
    for i in range(1,n+1):
        printString = ''
        if not(i % 3):
            printString += 'Fizz'
        if not(i % 5):
            printString += 'Buzz'
        if (i % 3) and (i % 5):
            printString += str(i)
        print(printString)

#main loop
while True:
    userInput = input('Hello. Please input the max number for your fizzbuzz: ').strip().lower()
    if userInput == 'q':
        break
    elif userInput == '' or not(userInput.isdigit()):
        print('Please input a number')
        continue
    else:
        fizz_buzz(int(userInput))
        continue

#test comment